tests/CMakeLists.txt

```diff
@@ -875,6 +875,7 @@ target_link_libraries(tests-offscreen ${OFFSCREEN_CTX_LIBRARIES})
 #
 # csgtexttest
 #
+find_package(OpenCSG REQUIRED)
```

Comment:
When adding this, we should remove the manual CSG finding code above as I assume they end up doing the same thing.